app/controllers/editions_controller.rb

```diff
@@ -3,12 +3,6 @@
 class EditionsController < ApplicationController
   def create
     result = Editions::CreateInteractor.call(params: params, user: current_user)
-    if result.draft_current_edition
-      # FIXME: this shouldn't be an exception but we've not worked out the
-      # right response - maybe bad request or a redirect with flash?
-      raise "Can't create a new edition when the current edition is a draft"
-    else
-      redirect_to edit_document_path(params[:document])
-    end
+    redirect_to edit_document_path(params[:document])
```

Comment:
I somewhat feel that this may obfuscates the underlying check that happens - i.e. it reads that the controller will always redirect to the "edit document" path and wouldn't be clear that users can get rerouted to just the "document" path